packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanel.tsx

```diff
@@ -567,7 +576,7 @@ export function TransferPanel() {
         }
       }
     },
-    [gasSummary, ethBalance, selectedToken]
+    [gasSummary, ethBalance, selectedToken, l2NetworkChainId]
```

Comment:
I think we're missing `isDepositMode` from the dependency array? Make sure you have this installed so it gives you a warning: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint